src/App.tsx

```diff
@@ -1,18 +1,27 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42)
-  .map(n => `Item ${n}`);
+const items = getNumbers(1, 42);
 
 export const App: React.FC = () => {
+  const [currentPage, changePage] = useState(1);
+  const [perPage, changePerPage] = useState(5);
+
+  const visibleItems = items
+    .filter((item) => {
+      return item <= currentPage * perPage
+      && item > currentPage * perPage - perPage;
+    });
```

Comment:
```suggestion
    .filter((item) => (
      item <= currentPage * perPage
        && item > currentPage * perPage - perPage;
    ));
```
You can also immediately return the value